src/Microsoft.IdentityModel.Tokens/Exceptions/SecurityTokenException.cs

```diff
@@ -66,6 +68,15 @@ protected SecurityTokenException(SerializationInfo info, StreamingContext contex
         {
         }
 
+        /// <summary>
```

Comment:
Can we fill this out?

Reply:
I'll update it tomorrow, thanks for pointing it out.

Reply:
Added.